libraries/botbuilder-dialogs/tests/dialogHelper.test.js

```diff
@@ -10,15 +10,9 @@ const {
     TestAdapter,
     UserState,
 } = require('botbuilder-core');
-const {
-    ComponentDialog,
-    DialogReason,
-    runDialog,
-    TextPrompt,
-    WaterfallDialog
-} = require('../');
+const { ComponentDialog, DialogReason, runDialog, TextPrompt, WaterfallDialog } = require('../');
 const { AuthConstants } = require('../lib/prompts/skillsHelpers');
-const { EndOfConversationCodes } = require('../../botbuilder-core')
+const { EndOfConversationCodes } = require('../../botbuilder-core');
```

Comment:
`botbuilder-core` is a dependency

```suggestion
const { EndOfConversationCodes } = require('botbuilder-core');
```